src/testing/README.md

```diff
@@ -0,0 +1,26 @@
+# @o1js/testing
+
+This is package contains testing utils used internally in o1js to test primitives.
+
+## Installation
+
+```bash
+npm install --save-dev @o1js/testing
+# or with yarn:
+yarn add -D @o1js/testing
+```
```

Comment:
I would remove `yarn`, its v1 version has been deprecated for years now and v2+ never took off